chapters/publication.tex

```diff
@@ -205,43 +201,44 @@ \subsection{Technical writing with \LaTeX}
 
 \subsection{Getting started with \LaTeX\ in the cloud}
 
 \LaTeX\ is a challenging tool to get started using,
 but the control it offers over the writing process is invaluable.
 In order to make it as easy as possible for your team
 to use \LaTeX\ without all members having to invest in new skills,
-we suggest using a web-based implementation as your first foray into \LaTeX\ writing.
+we suggest using a cloud-based implementation as your first foray into \LaTeX\ writing.
 Most such sites offer a subscription feature with useful extensions and various sharing permissions,
 and some offer free-to-use versions with basic tools that are sufficient
 for a broad variety of applications,
 up to and including writing a complete academic paper with coauthors.
 
-Cloud-based implementations of \LaTeX\ are suggested here for several reasons.
-Since they are completely hosted online,
-they avoids the inevitable troubleshooting of setting up a \LaTeX\ installation
-on various personal computers run by the different members of your team.
-They also typically maintain a single continuously synced master copy of the document
+Cloud-based implementations of \LaTeX\ have several advantageous features.
+First, since they are completely hosted online,
+they avoid the inevitable troubleshooting required to set up a \LaTeX\ installation
+on various personal computers run by the different members of a team.
+Second, they typically maintain a single, continuously synced, master copy of the document
 so that different writers do not create conflicted or out-of-sync copies,
 or need to deal with Git themselves to maintain that sync.
-They typically allow inviting collaborators to edit in a fashion similar to Google Docs,
+Third, they typically allow collaborators to edit in a fashion similar to Google Docs,
 though different services vary the number of collaborators and documents allowed at each tier.
-Most importantly, some tools provide a ``rich text'' editor
+Fourth, and most usefully, some implementations provide a ``rich text'' editor
 that behaves pretty similarly to familiar tools like Word,
 so that collaborators can write text directly into the document without worrying too much
 about the underlying \LaTeX\ coding.
 Cloud services also usually offer a convenient selection of templates
 so it is easy to start up a project and see results right away
 without needing to know a lot of the code that controls document formatting.
 
-On the downside, there is a small amount of up-front learning required,
+Cloud-based implementations of \LaTeX\ also have disadvantages.
+There is a small amount of up-front learning required,
 continous access to the Internet is necessary,
 and updating figures and tables requires a bulk file upload that is tough to automate.
-One of the most common issues you will face using online editors will be special characters
+A common problem you will face using online editors is special characters
```

Comment:
This sentence is too granular to fit in the book IMHO. And if we should keep it we should not keep it in a cloud specific section. We could perhaps move it next to `what-you-see-is-what-you-get' around line 127, as an example when what you see is what you get does not apply.

Reply:
I'm fine with moving this to a wiki article

Reply:
I agree. I think we can remove the next 6 lines. This is true of all LaTeX use, no matter if it's in the cloud or not.

Reply:
move to wiki or to our latex training

Reply:
done in cc13a138fc94fd7df3dc6a9af8047e88ec7a3a8b